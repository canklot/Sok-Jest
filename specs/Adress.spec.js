import TestComponent from '../components/Adres.vue'
import { mount } from '@vue/test-utils'

test('mount a vue component', () => {
    const wrapper = mount(TestComponent
    )
    expect(wrapper).toMatchSnapshot()
  })