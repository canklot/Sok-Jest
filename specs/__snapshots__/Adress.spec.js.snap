// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue component 1`] = `
<div class="Adres"><img src="../images/ucgen.png" class="ucgen">
  <div class="SilikAdres">Adress Seçin</div> <button class="MaviAdres">
    <p class="AdressSec">Adres Seç</p>
    <p class="ok">›</p>
  </button>
  <div class="SagTaraf">
    <div class="griArkaplan"><img src="../images/bisiklet.png" class="bisiklet">
      <div class="adreseTeslim">Adrese Teslim</div>
      <div class="minSepet">Min sepet</div>
      <div class="gonderim">Gönderim</div> <img src="../images/market.png" class="market">
      <div class="gelAl">Marketten Gel Al</div>
      <div class="minSepet">Min sepet</div>
    </div>
  </div>
</div>
`;
